Makefile

```diff
@@ -25,8 +25,9 @@ test:
 # Generate a basic test coverage report. This will open the report in your
 # browser. See also https://codecov.io/gh/elves/elvish/.
 cover:
-	go test -covermode=set -coverprofile=$@ ./...
-	go tool cover -html=$@
+	go test -coverprofile=cover -coverpkg=./pkg/... ./pkg/...
+	go tool cover -html=cover
+	go tool cover -func=cover | tail -1 | awk '{ print "Overall coverage:", $$NF }'
```

Comment:
Why change `$@` to `cover`?

Reply:
For consistency with the other coverage generators in *.cirrus.yml* and *.appveyor.yml*. This also makes it easier to copy/paste the statements without someone having to know what `$@` means in a makefile context.

Reply:
Sounds good.